### PR TITLE
Show uncle blocks

### PR DESCRIPTION
**Pull request by mvaivre**
To be released after web3 1.0.2 is out of rc.

### CODE REVIEW

File: apps/desktop-wallet/turbo.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "extends": ["//"],
-  "pipeline": {
+  "tasks": {
```
Comment (nop33) [inline apps/desktop-wallet/turbo.json:3]: What is this change for?

File: apps/explorer/src/api/assets/assetsApi.ts [hunk 2]
```diff
@@ -72,54 +72,58 @@ const NFTCollectionsMetadata = create({
...
+      queryOptions({
+        queryKey: ['verifiedTokens', network],
+        queryFn: (): Promise<VerifiedFungibleTokenMetadata[]> => {
+          try {
+            return fetch(`https://raw.githubusercontent.com/alephium/token-list/master/tokens/${network}.json`).then(
+              (r) => r.json().then((j: TokenList) => j.tokens.map((v) => ({ ...v, type: 'fungible', verified: true })))
+            )
+          } catch (e) {
+            console.error(e)
+            return Promise.reject(new Error(i18n.t('Verified token fetch failed')))
```
Comment (nop33) [inline apps/explorer/src/api/assets/assetsApi.ts:96]: I know you didn't make this change, but we shouldn't call it "Verified" anymore

File: apps/explorer/src/pages/BlockInfoPage.tsx [hunk 3]
```diff
@@ -311,3 +287,9 @@ const IODetailList = styled.div`
   border-radius: 8px;
   padding: 15px;
 `
+
+const UncleBlocks = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+`
```
Comment (nop33) [inline apps/explorer/src/pages/BlockInfoPage.tsx:295]: Seems like a formatting error here

**Review by nop33 — COMMENTED** [review]
First pass, I'll do a more in-depth review tmr

File: apps/desktop-wallet/turbo.json (hunk 1 above)
Comment (mvaivre) [reply apps/desktop-wallet/turbo.json:3]: New pnpm version.

File: apps/explorer/src/api/assets/assetsApi.ts (hunk 2 above)
Comment (mvaivre) [reply apps/explorer/src/api/assets/assetsApi.ts:96]: yep

File: apps/explorer/src/pages/BlockInfoPage.tsx (hunk 3 above)
Comment (mvaivre) [reply apps/explorer/src/pages/BlockInfoPage.tsx:295]: Why? It's just the end line indicator I believe?

**Review by nop33 — COMMENTED** [review]
Can't run it, getting some type errors

image

File: apps/desktop-wallet/package.json [hunk 4]
```diff
@@ -59,7 +59,7 @@
     "@alephium/token-list": "^0.0.12",
     "@alephium/typescript-config": "workspace:*",
     "@alephium/walletconnect-provider": "1.0.5",
```
Comment (nop33) [inline apps/desktop-wallet/package.json:61]: Also needs to be updated ❇️

File: apps/mobile-wallet/package.json [hunk 5]
```diff
@@ -22,7 +22,7 @@
     "@alephium/shared-react": "workspace:*",
     "@alephium/token-list": "^0.0.12",
     "@alephium/walletconnect-provider": "1.0.5",
```
Comment (nop33) [inline apps/mobile-wallet/package.json:24]: Also needs to be updated

**Comment by nop33** [comment]
@mvaivre I would close this PR in favor of https://github.com/alephium/alephium-frontend/pull/703 which has your commits cherry-pick'ed

**Comment by mvaivre** [comment]
Superseded by #703

**Pull request closed without merging**